tests/config.c

```diff
@@ -46,6 +43,7 @@
 #define TEST_CONFIG_DEFAULT_NUM_NORMAL_BG_THREADS   0
 #define TEST_CONFIG_DEFAULT_NUM_MEMTABLE_BG_THREADS 0
 
+#define TEST_CONFIG_DEFAULT_QUEUE_SCALE_PERCENT (100)
```

Comment:
It will be helpful to document here what the expected value for this newly added `--queue-scale-percent` config param should be.

I am assuming the expected value is an integer within the range of [0 - 100]. If so, it will be helpful to document the expected range of valid values.

Reply:
Any number is valid.  No other parameter is documented.